pkg/types/v1beta1/cluster.go

```diff
@@ -67,6 +67,7 @@ type MountImage struct {
 	Labels     map[string]string `json:"labels,omitempty"`
 	Cmd        []string          `json:"cmd,omitempty"`
 	Entrypoint []string          `json:"entrypoint,omitempty"`
+	ImagePhase ImagePhase        `json:"imagePhase,omitempty"`
```

Comment:
This field seems unnecessary

Reply:
uncompleted pr, I'am considering if we can use this field to discribe image install status.